sdk/create-dapp/templates/react-client-dapp/src/WalletStatus.tsx

```diff
@@ -0,0 +1,21 @@
+import { useCurrentAccount } from "@mysten/dapp-kit";
+import { Container, Flex, Heading, Text } from "@radix-ui/themes";
+import { OwnedObjects } from "./OwnedObjects";
+
+export function WalletStatus() {
+  const account = useCurrentAccount();
+
+  return (
+    <Container my="2">
+      <Heading mb="2">Wallet Status</Heading>
+      {!account && <Text>Wallet not connected</Text>}
+      {account && (
```

Comment:
nit: a ternary would be nice instead of having these separate conditional checks